test/pkg/environment/common/setup.go

```diff
@@ -229,13 +229,13 @@ func (env *Environment) ForceCleanup(opts ...Option) {
 
 	// Delete all the nodes if they weren't deleted by the provisioner propagation
 	Expect(env.Client.DeleteAllOf(env, &v1.Node{},
-		client.HasLabels([]string{test.DiscoveryLabel}),
```

Comment:
Was this intended to be a different discovery label than the provisioner name label when we looked at doing concurrent tests in the same cluster?

Reply:
Yeah it's scoped to a test execution. Every object has this injected to it.

Reply:
Nodes don't have this label since they aren't created directly by the tests.